docs/csharp/language-reference/compiler-options/listed-alphabetically.md

```diff
@@ -59,7 +59,7 @@ The following compiler options are sorted alphabetically. For a categorical list
 |**/incremental**|Enables incremental compilation [obsolete].|  
 |[/keycontainer](../../../csharp/language-reference/compiler-options/keycontainer-compiler-option.md)|Specifies a strong name key container.|  
 |[/keyfile](../../../csharp/language-reference/compiler-options/keyfile-compiler-option.md)|Specifies a strong name key file.|  
-|[/langversion:\<string>](../../../csharp/language-reference/compiler-options/langversion-compiler-option.md)|Specify language version mode: ISO-1, ISO-2, 3, 4, 5, 6, or Default|  
+|[/langversion:\<string>](../../../csharp/language-reference/compiler-options/langversion-compiler-option.md)|Specify language version mode: ISO-1, ISO-2, 3, 4, 5, or Default|  
```

Comment:
I think version 7 should be added here, instead of removing version 6, because that's what the latest version of `csc` supports.